pytorch_lightning/trainer/trainer.py

```diff
@@ -529,6 +529,10 @@ def __init__(
         # Callback system
         self.on_init_end()
 
+    @property
+    def device(self) -> Union[None, str, object]:
```

Comment:
To me it is not clear, why the trainer should have such a property as well?

Reply:
it was there before, I just made it as read-only, but agree that it is strange

Reply:
I would remove it, it's not needed as far as I can tell, since now it has shifted over to the module